specs/stein/approved/amd-sev-libvirt-support.rst

```diff
@@ -0,0 +1,367 @@
+..
+ This work is licensed under a Creative Commons Attribution 3.0 Unported
+ License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+
+====================================================
+libvirt driver launching AMD SEV-encrypted instances
+====================================================
+
+https://blueprints.launchpad.net/nova/+spec/amd-sev
+
+This spec proposes a minimal set of work required in order for nova's
```

Comment:
s,a minimal set of,, 

from an upstream perspective we shouldn't argue what the "minimal work" is but what the "right amount of implementation is. minimalism vs keeping it simple.

Reply:
Not sure I entirely agree but OK, changed in next commit.